### PR TITLE
Convert Svg() to JSX

### PR DESCRIPTION
**Pull request by darkwing**
Ticking another one off.

### CODE REVIEW

**Comment by zaggy** [comment]
Hi, 

Maybe it is worth, while you are at it, to try an update `svg-inline-react` package to the latest version?

File: assets/images/Svg.js [hunk 1]
```diff
@@ -62,5 +62,5 @@ module.exports = function(name, props) {
     className = "";
   }
   props = Object.assign({}, props, { className, src: svg[name] });
-  return React.createElement(InlineSVG, props);
+  return <InlineSVG {...props} />;
```
Comment (wldcordeiro) [inline assets/images/Svg.js]: So this is an idea. But you'd need to touch more of the codebase. How about converting this component from:

```js
module.exports = function(name, props) {
  // eslint-disable-line
  if (!svg[name]) {
    throw new Error("Unknown SVG: " + name);
  }
  let className = name;
  if (props && props.className) {
    className = `${name} ${props.className}`;
  }
  if (name === "subSettings") {
    className = "";
  }
  props = Object.assign({}, props, { className, src: svg[name] });
  return <InlineSVG {...props} />;
};
```

to
```js
module.exports = function({ name, ...props }) {
  // eslint-disable-line
  if (!svg[name]) {
    throw new Error("Unknown SVG: " + name);
  }
  let className = name;
  if (props && props.className) {
    className = `${name} ${props.className}`;
  }
  if (name === "subSettings") {
    className = "";
  }
  props = { ...props, { className, src: svg[name] } };
  return <InlineSVG {...{ ...props, className, src: svg[name] }} />;
};
```

We would need to modify the usage of `Svg` but then we could use it like a component as well and convert the usages like so

```js
// from this
Svg("some-name", { foo: bar });

// to this
<Svg name="some-name", foo={bar} />
```

**Comment by wldcordeiro** [comment]
@zaggy I would leave it for now as it's a major version bump.

**Comment by darkwing** [comment]
@wldcordeiro I was thinking the same but I didn't want to every file that used it yet; easier if we JSX all the components and then make this change

**Comment by wldcordeiro** [comment]
>  I was thinking the same but I didn't want to touch every file that used it yet; easier if we JSX all the components and then make this change

@darkwing agreed. I'll amend the main issue with that as an item.

**Pull request merged**